### PR TITLE
Addressing Paul's DISCUSS

### PR DESCRIPTION
**Pull request by henkbirkholz**
by literally using his proposed phrasing as an addition to the secconsec.

### CODE REVIEW

File: draft-ietf-rats-uccs.md [hunk 1]
```diff
@@ -578,6 +578,7 @@ the secure channel and/or the application protocol (e.g., TLS and the
 protocol identified by ALPN) MUST specify the roles of the endpoints
 in a fashion that the security properties of conveying UCCS via a
 Secure Channel between the roles are well-defined.
+The Secure Channel cryptographic strength should be at least as strong as the cryptographic keys it is transporting through this Secure Channel.
```
Comment (thomas-fossati) [inline draft-ietf-rats-uccs.md]: avoid some prose redundancy
```suggestion
The cryptographic strength of the Secure Channel must be at least as strong as the cryptographic keys it transports.
```

Comment (henkbirkholz) [reply draft-ietf-rats-uccs.md]: poetry

**Review by thomas-fossati — APPROVED** [review]
really good stuff

**Comment by henkbirkholz** [comment]
aggreed

File: draft-ietf-rats-uccs.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-rats-uccs.md]: Why did the should change to a must?  That is wrong.
(Why must the strength be strong? That's just clumsy.)

**Comment by cabo** [comment]
(This was a duplicate of PR #35 and has been reverted in da9337e.)

**Pull request merged**